streamerbot/1.get-started/4.faq.md

```diff
@@ -174,5 +174,16 @@ In this example, if `%user%` doesn't exist, it will have the value `null` and th
 
 
 ### 21. Can I use math in a subaction?
-Yes, for most cases you will need a *Set Argument* subaction (`Core -> Arguments -> Set Argument`). In there you can use [inline functions](/guide/variables#inline-functions). Choose any variable name and then do the math in the *Value* field. If you wanted to convert seconds to milliseconds to then use it in a delay subaction, you can do it like this: `$math(%secondsVariable% * 1000)$`. Simply use the newly named variable in a *Delay* subaction afterwards. (Example: [see here](<https://i.imgur.com/8RoLBgN.png>))
+Yes, for most cases you will need a *Set Argument* subaction (`Core -> Arguments -> Set Argument`). In there you can use [inline functions](<https://wiki.streamer.bot/en/Inline-Functions>). Choose any variable name and then do the math in the *Value* field. If you wanted to convert seconds to milliseconds to then use it in a delay subaction, you can do it like this: `$math(%secondsVariable% * 1000)$`. Simply use the newly named variable in a *Delay* subaction afterwards. (Example: [see here](<https://i.imgur.com/8RoLBgN.png>))
 
+### 22. What are global variables?
```

Comment:
more over-arching questions like this should probably just link to proper docs guide page to reduce duplicate documentation - or maybe not even be in the FAQ? not entirely sure